src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const initalState = { categories: [] };
+
+// actions
+const CHECKSTATUS = 'CHECKSTATUS ';
```

Comment:
Same here, kindly make sure that the value of action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`